src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -109,20 +109,26 @@ private List<string> ValidateFilesToExclude(string basePath, string filesToExclu
 
                 foreach (var excludedFile in jsonExcludedFiles)
                 {
-                    var fullPath = Path.GetFullPath(Path.Combine(basePath, excludedFile));
+                    var platformFilePath = GetPlatformSupportedFilePath(excludedFile);
+                    var fullPath = Path.GetFullPath(Path.Combine(basePath, platformFilePath));
 
                     if (!File.Exists(fullPath))
                         _logger.LogWarning($"The specified file to exclude {fullPath} could not be found. Did you mean to exclude another file?");
 
                     excludedFiles.Add(fullPath);
                 }
             }
-            catch (Exception ex)
+            catch
             {
                 throw new ValidationException("Invalid JSON value provided for --files-to-exclude. The correct format, for example, should be: ['./ExampleClass.cs','./ExampleDirectory/ExampleClass2.cs','C:\\ExampleDirectory\\ExampleClass.cs'].");
             }
 
             return excludedFiles;
         }
+
+        private static string GetPlatformSupportedFilePath(string excludedFile)
```

Comment:
This could be usefull elsewhere too. Maybe the [FilePathUtils](https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.Core/Stryker.Core/FilePathUtils.cs) is a good place for this?